### ISSUE TITLE
Static client

### ISSUE THREAD

**Issue by 3kh0** [SoftbearStudios/mk48#207]
Is there a client that you can just upload to a Apache server and it will connect?

**Comment by finnbear** [comment]
Hi,

We usually recommend an `<iframe/>`-based solution. Even if the client could be served statically, it would break in every update.

**Comment by finnbear** [comment]
In fact we have a ready-made HTML file for this purpose: https://raw.githubusercontent.com/SoftbearStudios/mk48/main/assets/embed/iframe.html

**Comment by 3kh0** [comment]
@finnbear iframes do not work for my purpose

**Comment by finnbear** [comment]
Just took a peek at your project and it seems you are in the business of "unblocking" games. Depending on how the blocking is implemented, any connection to our official domain (whether from a copy of or iframe to our client) may be blocked. If we ever develop a single player version of Mk48.io (which we had in the past but didn't include in our port from Go to Rust), that might be more aligned with what you are trying to accomplish 🙂

**Comment by 3kh0** [comment]
@finnbear Yes, I first want to get a stand-alone client for the blocks that only detect the URL of the page, then later I can proxy the requests.
